### ISSUE TITLE
Feature: PR Title contains Issue number

### ISSUE THREAD

**Issue by fragsalat** [zalando/zappr#513]
We would like to enforce our PR titles to follow the same pattern as the commits because if you make a squash merge the PR title is by default taken for the commit message which often leads to a failing commit message check which is only fixable via rebasing. A PR title would be much easier to fix.

Would you like to have such a feature and merge a PR for this?

**Comment by harti2006** [comment]
@fragsalat Please note that the commit message check is not longer required to meet compliance requirements. Only approval + specification check. Do you in this case still need this feature?
Usually you get a proper "link" between PRs and issues, when you write "fixes #ISSUE_NUMBER" in the PR body.

**Comment by fragsalat** [comment]
Yeah because we decided to go with squash merges and feature and task branches and the PR title is used for the Commit message. So when you have a PR from a task into a feature branch with a pr title which doesn't match the commig message check the PR from feature to master branch will fail. Therefore I would like to have a check for the PR title as well.
